imports/ui/annotation.coffee

```diff
@@ -1,5 +1,7 @@
-annotateContent = (content, annotations, options={})->
-  { startingIndex, endingIndex, selectedAnnotationId } = options
+export annotateContent = (content, annotations, options={}) ->
+  { startingIndex, endingIndex, selectedAnnotationId, pointerEvents} = options
+  pointerEvents ?= true
```

Comment:
@nathanathan Oh, I see. Sorry about the confusion. How does it look now?
https://github.com/ecohealthalliance/eidr-connect/blob/45fff10fa4ad6fa293915e8a71fd323f03268f21/imports/ui/annotation.coffee